ironfish/src/memPool/feeEstimator.ts

```diff
@@ -4,43 +4,54 @@
 import { Assert } from '../assert'
 import { Blockchain } from '../blockchain'
 import { createRootLogger, Logger } from '../logger'
-import { MemPool, PriorityQueue } from '../memPool'
+import { MemPool } from '../memPool'
 import { getTransactionSize } from '../network/utils/serializers'
 import { Block, Transaction } from '../primitives'
 import { Wallet } from '../wallet'
 import { Account } from '../wallet/account'
 
-const SPEND_SERIALIZED_SIZE_IN_BYTE = 388
-const NOTE_SERIALIZED_SIZE_IN_BYTE = 467
-
 interface FeeRateEntry {
   feeRate: bigint
   blockHash: Buffer
 }
 
+export type PriorityLevel = typeof PRIORITY_LEVELS[number]
+export type Percentile = typeof PRIORITY_LEVEL_PERCENTILES[number]
+
+export const PRIORITY_LEVELS = ['low', 'medium', 'high'] as const
+const PRIORITY_LEVEL_PERCENTILES = [10, 20, 30] as const
+const PERCENTILES_TO_PRIORITY_LEVELS = new Map<Percentile, PriorityLevel>([
+  [10, 'low'],
+  [20, 'medium'],
+  [30, 'high'],
+])
+const PRIORITY_LEVELS_TO_PERCENTILES = new Map<PriorityLevel, Percentile>([
+  ['low', 10],
+  ['medium', 20],
+  ['high', 30],
+])
+const SPEND_SERIALIZED_SIZE_IN_BYTE = 388
+const NOTE_SERIALIZED_SIZE_IN_BYTE = 467
+
 export class FeeEstimator {
-  private queue: Array<FeeRateEntry>
+  private queues: Map<PriorityLevel, Array<FeeRateEntry>>
   readonly chain: Blockchain
   private wallet: Wallet
   private readonly logger: Logger
   private numOfRecentBlocks = 10
-  private numOfTxSamples = 3
-  private maxQueueLength: number
   private defaultFeeRate = BigInt(1)
+  private readonly percentiles = PRIORITY_LEVEL_PERCENTILES.map((x) => x)
```

Comment:
Do we need `percentiles` or can we use the `PRIORITY_LEVEL_PERCENTILES` constant?